data/ecosystems/s/solana.toml

```diff
@@ -321,6 +322,7 @@ github_organizations = [
   "https://github.com/firedancer-io",
   "https://github.com/generatus",
   "https://github.com/GenesysGo",
+  "https://github.com/Grim-Syndicate",
```

Comment:
this line can be removed, out ETL takes care of rolling all up

Reply:
done